command.go

```diff
@@ -24,9 +24,10 @@ func (cmd *Command) Parse(fields []interface{}) (err error) {
 		return errors.New("No enough arguments")
 	}
 
-	var ok bool
-	if cmd.Mechanism, ok = fields[0].(string); !ok {
-		return errors.New("Compression mechanism name must be a string")
+	if mechanism, ok := fields[0].(imap.RawString); !ok {
```

Comment:
`RawString` is never returned by the parsing code, it's just a hint for the formatting code. We should probably document that.

So we should keep using `string` here.

Reply:
Great then its even easier :)